docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -543,7 +543,7 @@ endif::[]
 
 .Procedure
 
-. In the upper-right corner of the {product-kafka} web console, select the gear icon, and click *Settings* > *User Access* > *Groups*
+. In the upper-right corner of the {product-kafka} web console, select the gear icon, and click *Settings* > *Identity & Access Management* > *User Access* > *Groups*
```

Comment:
```suggestion
. In the toolbar of the {product-kafka} web console, select the gear icon, and click *Settings* > *Identity & Access Management* > *User Access* > *Groups*.
```

Reply:
@smccarthy-ie I do not think we need to click "Settings" anymore?
Also, I think both guides should read the same first line and maybe Kafka should also say 
"At the top of the [Streams for Apache Kafka web console](https://console.redhat.com/application-services/streams/kafkas), select the gear icon."

Reply:
@fmcdonal Good spot! Yes should be "click *Identity & Access Management* > *User Access* > *Groups*" :)